src/redux/home.js

```diff
@@ -40,59 +41,29 @@ const homeReducer = (state = initialState, action) => {
   switch (action.type) {
     case GET_COUNTRY:
       return { ...state, covid_data: action.payload };
-
-      // case CLEAR_COUNTRY:
-      //   return state.map((mission) => (mission.id !== action.id ? mission
-      //     : { ...mission, join: !mission.join }));
-
-    case FILTER_REGION: {
-      const country = state.filter((countryDetail) => countryDetail.id === action.id);
-      return [country[0]];
-    }
     default:
       return state;
   }
 };
 
 export const loadCountryDetail = () => {
   const request = Countries.map(async (country) => {
-    // const array = [];
-    // let outgoingData;
     const data = await axios
       .get(`${baseUrl}/${country.name}`);
-      // then((response) => {
-      // });
-    const { '2020-03-10': dates } = data.data.dates;
-    console.log(dates.countries);
+    const { '2020-10-10': dates } = data.data.dates;
     const openData = Object.entries(dates.countries).map((object) => object[1]);
-    console.log(openData);
     const outgoingData = {
       id: uuidv4(),
       data: openData,
       country: country.name,
+      region: country.region,
+      imgUrl: country.img,
 
     };
     return outgoingData;
   });
-  console.log(request);
+  // console.log(request);
```

Comment:
### Nice to have
It would be nice to always delete commented lines in the codebase that no longer serve a purpose.